qt/python/mantidqt/mantidqt/widgets/helpwindow/helpwindowbridge.py

```diff
@@ -0,0 +1,71 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import argparse
```

Comment:
argparse is the one exception to imports at the top of a file. Moving it to inside of `main()` would reduce what is imported upon parsing this file.